docker-compose/docker-compose.yaml

```diff
@@ -90,6 +138,7 @@ services:
       - 'JAEGER_SERVER_URL=http://jaeger:16686'
       - 'GITHUB_BASE_URL=http://github-proxy:3180'
       - 'PROMETHEUS_URL=http://prometheus:9090'
+      - 'NEW_MIGRATIONS=true'
```

Comment:
Is this env variable still relevant for the release?

Reply:
Good catch, this won't be required come release.